saleor/dashboard/product/views.py

```diff
@@ -30,9 +30,10 @@ def product_class_list(request):
         (pc.pk, pc.name, pc.has_variants, pc.product_attributes.all(),
          pc.variant_attributes.all())
         for pc in classes.object_list]
+
```

Comment:
I would remove this empty line (and in other views you've added it as well). It is useful in complex functions but in most of our views it's unnecessary.